### PR TITLE
Update Heroku notes

### PR DESCRIPTION
**Pull request by JPrevost**
- reflect ACM is fine unless the app is special
- add link to touchstone notes from heroku page
- add note to fixlib about including our standard billing account number

### CODE REVIEW

File: heroku.md [hunk 1]
```diff
@@ -223,15 +223,22 @@ registration:
 
 CNAME: [YOUR_DESIRED_REGISTRATION].mit.edu
 Target: [YOUR_DESIRED_REGISTRATION].mit.edu.herokudns.com
+Account number: please include our standard billing account number
```
Comment (gravesm) [inline heroku.md]: Is this the actual text that should appear here?

Comment (JPrevost) [reply heroku.md]: Yeah. Andy said we needed to include one which they have and he kept forgetting to include it so this note will remind them to do so. I didn't want to include the actual account number as we weren't sure if they were considered secret.

Comment (JPrevost) [reply heroku.md]: I guess it may be useful to explain that in the docs...

**Comment by JPrevost** [comment]
@gravesm I added a clarifying note about your question. Any other thoughts before I merge?

**Comment by gravesm** [comment]
👍

**Pull request merged**